lingua_franca/res/text/sv-se/date_time.json

```diff
@@ -0,0 +1,129 @@
+{
+  "decade_format": {
+    "1": {"match": "^\\d$", "format": "{x}"},
+    "2": {"match": "^1\\d$", "format": "{xx}"},
+    "3": {"match": "^\\d0$", "format": "{x0}"},
+    "4": {"match": "^[2-9]\\d$", "format": "{x0} {x}"},
+    "default": "{number}"
+  },
+    "hundreds_format": {
+    "1": {"match": "^\\d{3}$", "format": "{x_in_x00} hundra"},
+    "default": "{number}"
+  },
+    "thousand_format": {
+    "1": {"match": "^\\d00\\d$", "format": "{x_in_x000} tusen"},
+    "2": {"match": "^1\\d00$", "format": "{xx_in_xx00} hundra"},
+    "3": {"match": "^\\d{2}00$", "format": "{x0_in_x000} {x_in_x00} hundra"},
+    "4": {"match": "^(1\\d{3})|(\\d0\\d{2})$", "format": "{xx_in_xx00}"},
+    "5": {"match": "^\\d{4}$", "format": "{x0_in_x000} {x_in_x00}"},
+    "default": "{number}"
+  },
+    "year_format": {
+    "1": {"match": "^\\d\\d?$", "format": "{formatted_decade} {bc}"},
+    "2": {"match": "^\\d00$", "format": "{formatted_hundreds} {bc}"},
+    "3": {"match": "^\\d{3}$", "format": "{formatted_hundreds} {formatted_decade} {bc}"},
+    "4": {"match": "^\\d{2}00$", "format": "{formatted_thousand} {bc}"},
+    "5": {"match": "^\\d00\\d$", "format": "{formatted_thousand} {formatted_decade} {bc}"},
+    "6": {"match": "^\\d{2}0\\d$", "format": "{formatted_thousand} noll {formatted_decade} {bc}"},
+    "7": {"match": "^\\d{4}$", "format": "{formatted_thousand} {formatted_decade} {bc}"},
+    "default": "{year} {bc}",
+    "bc": "före kristus"
```

Comment:
Does Swedish also need to account for "C.E."? (English does)

Reply:
C. E. That's common era? There seem to exist a swedish version "vår tideräkning" and "före vår tideräkning" but I've never heard it used.

Reply:
Fair enough. Approved, if you see this before I get back to a computer. I'm on a phone, seems risky to merge =P